mup.js

```diff
@@ -0,0 +1,63 @@
+module.exports = {
```

Comment:
This file doesn't seem to be real. please delete this and the settings.json file

Reply:
The whole .deploy  shouldn't be there, I don't need you to merge this into the project, it's an error on my side, I removed the folder